Core/TImeSeries/TimeSeries.cs

```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Core.TimeSeries
+{
+	public class TimeSeries
+	{
+		protected Date End = Date.MinValue;
+
+		protected Date Start = Date.MaxValue;
+
+		protected List<double> Series { get; } = new List<double>();
+
+		public virtual double this[Date date]
+		{
+			get
+			{
+				if (date < Start || date > End)
+				{
+					return 0;
+				}
+
+				var index = (date - Start).Days;
+				return Series[index];
+			}
+		}
+
+		public virtual void AddMany(IEnumerable<Transaction> transactions)
+		{
+			transactions = transactions ?? throw new ArgumentNullException(nameof(transactions));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use the '??=' operator here.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4992784)